plan.go

```diff
@@ -247,40 +251,28 @@ func extractSelectionSet(ctx *PlanningContext, insertionPoint []string, parentTy
 					continue
 				}
 				implementationType := ctx.Schema.Types[implementationName]
-				possibleId := &ast.InlineFragment{
-					TypeCondition: implementationName,
-					SelectionSet: []ast.Selection{
-						&ast.Field{
-							Alias:      "_id",
-							Name:       "id",
-							Definition: implementationType.Fields.ForName("id"),
-						},
-					},
-					ObjectDefinition: implementationType,
+
+				if idDef := implementationType.Fields.ForName("id"); idDef != nil {
+					possibleId := &ast.InlineFragment{
+						TypeCondition: implementationName,
+						SelectionSet: []ast.Selection{&ast.Field{Alias: "_bramble_id", Name: "id", Definition: idDef}},
```

Comment:
Could pay to pull `_bramble_id` out into a const since it's used in a few spots.